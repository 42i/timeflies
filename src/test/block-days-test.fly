day 2012-09-08 8 16
; ha ha
leave-days 2012-09-09 2012-09-14 test-leave
; ho ho
day 2012-09-15 8 16
; hee hee
sick-days 2012-09-17 2012-09-21 test-sickness
; hi ho
