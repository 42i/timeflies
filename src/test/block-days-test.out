Time at work overview (all):
     when        worked   leave    sick balance
2012-09-08 Sat:    8.00 ----.-- ----.--    8.00
              ; ha ha
              ; ho ho
  week 2012-36:    8.00 ----.-- ----.--    8.00
2012-09-10 Mon: ----.--    8.00 ----.-- ----.-- test-leave
2012-09-11 Tue: ----.--    8.00 ----.-- ----.-- test-leave
2012-09-12 Wed: ----.--    8.00 ----.-- ----.-- test-leave
2012-09-13 Thu: ----.--    8.00 ----.-- ----.-- test-leave
2012-09-14 Fri: ----.--    8.00 ----.-- ----.-- test-leave
2012-09-15 Sat:    8.00 ----.-- ----.--    8.00
              ; hee hee
              ; hi ho
  week 2012-37:    8.00   40.00 ----.--    8.00
2012-09-17 Mon: ----.-- ----.--    8.00 ----.-- test-sickness
2012-09-18 Tue: ----.-- ----.--    8.00 ----.-- test-sickness
2012-09-19 Wed: ----.-- ----.--    8.00 ----.-- test-sickness
2012-09-20 Thu: ----.-- ----.--    8.00 ----.-- test-sickness
2012-09-21 Fri: ----.-- ----.--    8.00 ----.-- test-sickness
  week 2012-38: ----.-- ----.--   40.00 ----.--
 month 2012-09:   16.00   40.00   40.00   16.00
         total:   16.00   40.00   40.00   16.00
     when        worked   leave    sick balance
